.github/workflows/ci-linux.yml

```diff
@@ -0,0 +1,139 @@
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL VMWARE, INC. OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+# OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+# DAMAGE.
+
+
+# Github Actions workflow for Continuous Integration testing on Linux.
+# TODO i#4549: This is a work-in-progress and not enabled currently.
+
+name: ci-linux
+on:
+  # Run on pushes to any repository branch.
+  # TODO i#4549: Uncomment when Github Actions integration complete.
+  # push:
+
+  # Manual trigger using the Actions page. May remove when integration complete.
+  workflow_dispatch:
+
+jobs:
+  x86:
+    runs-on: ubuntu-16.04
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: master
+
+    - name: Create Build Environment
+      run:
+        sudo apt-get -y install doxygen vera++ cmake zlib1g-dev libsnappy-dev;
+        sudo apt-get -y install g++-multilib;
+
+    - name: Run Suite
+      working-directory: ${{ github.workspace }}
+      shell: bash
+      run: ./suite/runsuite_wrapper.pl travis $EXTRA_ARGS
+      env:
+        DYNAMORIO_CROSS_AARCHXX_LINUX_ONLY: no
+        DEPLOY: no
+        EXTRA_ARGS: "32_only"
+
+  x86-64:
+    runs-on: ubuntu-16.04
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: master
+
+    - name: Create Build Environment
+      run:
+        sudo apt-get -y install doxygen vera++ cmake zlib1g-dev libsnappy-dev;
+        sudo apt-get -y install g++-multilib;
+
+    - name: Run Suite
+      working-directory: ${{ github.workspace }}
+      shell: bash
+      run: ./suite/runsuite_wrapper.pl travis $EXTRA_ARGS
+      env:
+        DYNAMORIO_CROSS_AARCHXX_LINUX_ONLY: no
+        DEPLOY: no
+        EXTRA_ARGS: "64_only"
+
+  archxx-cross-compile:
+    runs-on: ubuntu-16.04
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: master
+
+    - name: Create Build Environment
+      run:
+        sudo apt-get -y install doxygen vera++ cmake zlib1g-dev libsnappy-dev;
+        sudo apt-get -y install g++-arm-linux-gnueabihf g++-aarch64-linux-gnu;
+
+    - name: Run Suite
+      working-directory: ${{ github.workspace }}
+      shell: bash
+      run: ./suite/runsuite_wrapper.pl travis $EXTRA_ARGS
+      env:
+        DYNAMORIO_CROSS_AARCHXX_LINUX_ONLY: yes
+        DEPLOY: no
+
+  android-arm-cross-compile:
+    runs-on: ubuntu-16.04
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: master
+
+    - name: Create Build Environment
+      # Manually force using ld.bfd, setting CMAKE_LINKER does not work.
```

Comment:
This comment makes more sense down by where we do this.

Reply:
Done. I had added it here before when I hadn't discovered the multi-line run syntax. But inlining the comment seems to work now.